### PR TITLE
Add resource recommendations for SCIM and Redis deployments to README

### PR DESCRIPTION
**Pull request by chasdl**
## Overview

Over the course of the last few months we've updated the requests/limits for our SCIM bridge and Redis deployments. These resource recommendations are acceptable in most scenarios, but they fall short in high volume deployments.

Here we provide a set of recommendations specifically in cases where the SCIM bridge is deployed for a customer that has a high number of users and/or groups.

## Changes

* Additions to the README specifying resource recommendations.

-----------------------------------------------------------------------

## Checklist

- [X] review the [guide to contributing](https://github.com/1Password/op-scim-helm/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,36 @@ helm uninstall my-release
...
+  memory: 512M
+
+limits:
+  cpu: 1 (1000m)
+  memory: 1024M
+```
+
+This is 4x the CPU, and 2x the memory.
+
+Please reach out to our support team if you need help with the configuration or to tweak the values for your deployment.
```
Comment (devillexio) [inline README.md:66]:
```suggestion
Please reach out to our [support team](https://support.1password.com/contact/) if you need help with the configuration or to tweak the values for your deployment.
```

File: README.md [hunk 2]
```diff
@@ -31,4 +31,36 @@ helm uninstall my-release
...
+requested:
+  cpu: 0.5 (500m)
+  memory: 512M
+
+limits:
+  cpu: 1 (1000m)
+  memory: 1024M
+```
+
+This is 4x the CPU, and 2x the memory.
```
Comment (devillexio) [inline README.md:64]:
```suggestion
This proposal is 4x the CPU and 2x the memory of the default values.
```

File: README.md [hunk 3]
```diff
@@ -31,4 +31,36 @@ helm uninstall my-release
 
 ## Available charts
 
-* [1Password SCIM bridge](https://github.com/1Password/op-scim-helm/tree/main/charts/op-scim-bridge)
\ No newline at end of file
+* [1Password SCIM bridge](https://github.com/1Password/op-scim-helm/tree/main/charts/op-scim-bridge)
+
+## Resource Recommendations
```
Comment (devillexio) [inline README.md:36]: What do you think about:
1. providing instructions and/or an example on how you can override these values in Helm commands, and
2. link to the Kubernetes docs that describe how these resource units are measured?

**Review by devillexio — COMMENTED** [review]
Looks good overall. 🙌 

Left a couple of minor suggestions.

Please make a note to also update the README that is shown when you navigate to the [repo URL](https://1password.github.io/op-scim-helm) in a browser, i.e.:
```
❯ helm repo list
NAME            	URL                                     
bitnami         	https://charts.bitnami.com/bitnami      
1password       	https://1password.github.io/op-scim-helm
```

This README is maintained on a special branch: [gh-pages](https://github.com/1Password/op-scim-helm/tree/gh-pages). Note that you should not PR into this branch, or merge this branch into main. It's used mostly by the release automation action run in the pipeline. The README is a quality of life improvement for folks that happen to navigate to the HELM repo URL in the browser.

Shout if you need help. 🙂

File: README.md (hunk 3 above)
Comment (chasdl) [reply README.md:36]: Agree!

**Review by devillexio — APPROVED** [review]
LGTM! :tada:

**Pull request merged**